packages/shared/src/modules/extension-bridge/ExtensionBridge.ts

```
@@ -0,0 +1,108 @@
+
+  static requestPageContent() {
+    return Tab.sendMessage<void, { content: string; category: Category }>(BRIDGE_MESSAGE_TYPES.PAGE_CONTENT);
+  }
+
+  private static checkYoutube(url: string) {
+    const youtubeRegex = /^(https?:\/\/)?(www\.)?(youtube\.com|youtu\.be)\/.+$/;
+    return youtubeRegex.test(url);
+  }
+
+  private static getCategory(url: string): Category {
+    if (this.checkYoutube(url)) return 'youtube';
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)